4337/contracts/test/SafeMock.sol

```diff
@@ -164,7 +163,8 @@ contract Safe4337Mock is SafeMock {
         _validateSignatures(entryPoint, userOp);
 
         if (requiredPrefund != 0) {
-            entryPoint.call{value: requiredPrefund}("");
+            (bool success, ) = entryPoint.call{value: requiredPrefund}("");
+            success;
```

Comment:
Suppress the Solidity compiler warning that "result from a low level call was not used".